.github/workflows/generated-linux-aarch64-binary-manywheel-nightly.yml

```diff
@@ -54,7 +54,6 @@ jobs:
       ALPINE_IMAGE: "arm64v8/alpine"
       build_name: manywheel-py3_8-cpu-aarch64
       build_environment: linux-aarch64-binary-manywheel
```

Comment:
This is wrong we need the PYTORCH_EXTRA_INSTALL_REQUIREMENTS here. We want metadata for all the wheels be identical.